index.html

```diff
@@ -0,0 +1,187 @@
+        <div> <img src="images/Portfolio_4.png" class="img_2"> </div>
+        <div class="section_1">
+         <h2 class="head_2"> Multi-Post <br> Stories </h2>
+         <ul class="list-ul">
+           <li class="list_1"> CANOPY </li>
+           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
+           <li class="list_2"><img src="images/Counter_2.png">2015</li>
+               </ul>
+         <p>A daily selection of privately
+            personalized reads; no accounts 
+            or sign-ups required.</p>   
+         <button class="btn_3"> html</button>
+         <button class="btn_4">css</button>
+         <button class="btn_5">javascript</button>
+         </div>
+        <button class="btn_2">See Project</button>
+      </div>
+
+  </div>
+</section>
+
+     <div class="container_2">
+        <section class = "section_2">
+          <h2> About <br> Myself</h2>
+          <p> I’m a software developer! I can help you build a product,
+            feature or website Look through some of my
+            work and experience! If you like what you see and
+             have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <a href="#" class="connect_1"> LET'S CONNECT</a>
+          <ul class="social_2">
+            <li><a href="https://github.com/david30-maker"><img src="images/github1.png" alt="github logo"></a></li>
+            <li><a href="https://www.linkedin.com/in/david-igbo-464196184"><img src="images/Linkedin_1.png" alt="in logo"></a></li>
+            <li><a href="https://venture.angellist.com/v/begin"><img src="images/angellist.png" alt="angel logo"></a></li>
+            <li><a href="https://twitter.com/@okpahdavid"><img src="images/twitter_1.png" alt="Twiiter logo"></a></li>
+            <li><a href="#"><img src="images/medium.png" alt="m logo"></a></li>
+          </ul>
+        </section>
+    
+        <button class="btn"> Get my resume </button>
+    
+        <div class="div-7">
+        <ul>
+          <li><a href="#" class="lan-1">Languages</a></li>
+          <li class="faa-1"><a href="#">
+            <i class="fa fa-angle-down"></i></a></li>
+        </ul>
+          </div>
+
+         <div id="faa-2">
+        <ul>
+          <li><a href="#">JavaScript</a></li>
+          </ul>
+          </div>
+
+          <div class="div-2">
+            <ul>
+          <li><a href="#">HTML</a></li>
+           </ul>
+          </div>
+
+          <div class="div-3">
+            <ul>
+          <li><a href="#">CSS</a></li>
+           </ul>
+          </div>
+
+          <div class="div-5">
+            <ul>
+            <li><a href="#">Frameworks</a></li> 
+            <li class="link1"><a href="#">❯</a></li> 
+            </ul>
+          </div> 
+          <hr>
+
+          <div class="div-6">
+            <ul>
+              <li><a href="#">Skills</a></li> 
+              <li class="link2"><a href="#">❯</a></li> 
+            </ul>
+          </div>
```

Comment:
[OPTIONAL] Great job on using `ul` and `li` as part of the project requirements 👍🏽 but your design seems to be overflowing horizontally, make sure to address that in order to make it look better.See below 👇🏽 

## Your design
![Screenshot 2023-01-19 at 08 26 59](https://user-images.githubusercontent.com/45739963/213454616-d2ff88e7-910d-4b16-9e4a-fd99d088cb9e.png)

there are several ways to achieve that, you could try setting a limit to the container of this section for example, you could try setting an overflow limit. These are just ideas and the implementations might change other parts of your code so be sure to consider that. 👀